core/src/main/scala/chisel3/package.scala

```diff
@@ -385,4 +385,7 @@ package object chisel3 {
   // Connection exceptions.
   case class BiConnectException(message: String) extends ChiselException(message)
   case class MonoConnectException(message: String) extends ChiselException(message)
+
+  final val deprecatedMFCMessage =
```

Comment:
Can/Should this be package private?

Reply:
I was trying to hide this, but `@deprecated` seemed to elide the message if it was package private. I'll check.